specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/ipAddressManager.json

```diff
@@ -0,0 +1,1119 @@
+            "description": "Optional key by which to sort.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "sortValue",
+            "description": "Optional sort value for pagination.",
+            "type": "string"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of IpamPools",
+            "schema": {
+              "$ref": "#/definitions/IpamPoolList"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CommonErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "IpamPools_List": {
+            "$ref": "./examples/IpamPools_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/ipamPools/{poolName}": {
+      "put": {
+        "tags": [
+          "IpamPools"
+        ],
+        "summary": "Creates/Updates the Pool resource.",
+        "operationId": "IpamPools_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "poolName",
+            "description": "IP Address Manager Pool resource name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9-]*$"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Pool resource object to create/update.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IpamPool"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Pool was updated",
+            "schema": {
+              "$ref": "#/definitions/IpamPool"
+            }
+          },
+          "201": {
+            "description": "Pool was created",
+            "schema": {
+              "$ref": "#/definitions/IpamPool"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
```

Comment:
compared to 18922 from private this was added only in this pr

Reply:
Yes, this was added for ipAddressManager after the private PR. This added to support long running PUTs for IpamPools resource.